Bottom: 7a2aaf2e7010b0c7fdaadc0b8a6be7a0140caca2
Top:    0cb7fe25c6adb6aa58bd08520a47a094ba7dc14d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-03 23:13:09 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
